### PR TITLE
add doc for policy framework and controllers

### PR DESCRIPTION
**Pull request by ycao56**
https://github.com/open-cluster-management/backlog/issues/11335

### CODE REVIEW

File: content/getting-started/install-policy-controllers.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+```Shell
+git clone https://github.com/open-cluster-management/config-policy-controller.git
+```
+
+Deploy the `config-policy-controller` to the managed cluster. 
+
+```Shell
+export MANAGED_CLUSTER_NAME=<managed cluster name> # export MANAGED_CLUSTER_NAME=cluster1
+kubectl config use-context kind-cluster1
```
Comment (mikeshng) [inline content/getting-started/install-policy-controllers.md:35]: from your other file: `kubectl config use-context kind-$MANAGED_CLUSTER_NAME` seems a bit better.

File: content/getting-started/install-policy-controllers.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+...
+```
+
+## Install certificate policy controller
+
+TODO
+
+## Install IAM policy controller
+
+TODO
```
Comment (mikeshng) [inline content/getting-started/install-policy-controllers.md:56]: After installation of the framework and the controller, are there enough pieces now for a small policy sample that the user can deploy and see it in action?

File: content/getting-started/install-policy-controllers.md (hunk 1 above)
Comment (ycao56) [reply content/getting-started/install-policy-controllers.md:35]: I will update. Was trying to follow cluster and app

Comment (ycao56) [reply content/getting-started/install-policy-controllers.md]: done

File: content/getting-started/install-policy-controllers.md (hunk 2 above)
Comment (ycao56) [reply content/getting-started/install-policy-controllers.md]: I will add `## What is next`

**Comment by mikeshng** [comment]
@qiujian16 the changes look good to me. As we are planning to present the policy piece to the community this week, could you please take a look and have this PR merge before the other ones? Thanks.

**Pull request merged**